util/compliance.py

```diff
@@ -0,0 +1,84 @@
+# This util searches a standards document against our procedure docs and returns likely
+# matches for each standard
+# Run with Python3!!
+
+
+from json import load
+from os import path, rename, mkdir, listdir
+from nltk.corpus import stopwords
+from nltk.tokenize import word_tokenize
+from nltk import download
+from whoosh.index import create_in, open_dir
+from whoosh.fields import Schema, TEXT
+from whoosh.qparser import QueryParser
+from whoosh import scoring
+
+directory = '/Users/jpdimarz/Repos/security-policy-templates/templates/procedures' #set this to the appropriate path
+standards_file = 'csa-ccm.json' #set this to the appropriate file
+#download('punkt')  these must be downloaded the first time the script is ran
+#download('stopwords')
```

Comment:
Perhaps something like [`asdf`](https://github.com/danhper/asdf-python) could be encouraged for Python version management, and then `pip` for dependencies, or whatever the latest best practices are? It would be nice to avoid having to read this source code to find these comments and get something installed by uncommenting, then re-commenting etc.

Reply:
I added a script that uses pip to install all dependencies. I looked into Python packaging via setuptools and it's somewhat of a process, so I don't think it's necessary considering it's just a util script. If we develop it further or decide that it's useful enough I could package it properly.